component-runtime-impl/src/main/java/org/talend/sdk/component/runtime/record/RecordImpl.java

```diff
@@ -80,6 +82,8 @@ private RecordImpl(final Map<String, Object> values, final Schema schema) {
     @Override
     public <T> T get(final Class<T> expectedType, final String name) {
         final Object value = values.get(name);
+        // here mean get(Object.class, name) return origin store type, like DATETIME return long, DECIMAL return string,
+        // is expected?
```

Comment:
when call this:
```
get(Object.class, name)
```
it will return store value, for example, datetime is stored as a long value, than that return a long value, not sure that's expected.